core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -706,6 +706,23 @@ abstract class RDD[T: ClassTag](
   }
 
   /**
+   * Spark's internal mapPartitions method which skips closure cleaning. To be used carefully
+   * only if we are sure that the RDD elements are serializable and don't require closure
+   * cleaning
```

Comment:
can you add that this is mainly for performance improvements? Also you're missing a period at the end.